dist/components/fullpage.js

```diff
@@ -191,7 +201,7 @@ var Fullpage = function (_React$Component) {
       this.setState({ 'scrollPending': true });
 
       var self = this;
-      scrollTo(BODY, self.state.slides[activeSlide], 500, function () {
+      scrollTo(getBody(), self.state.slides[activeSlide], 500, function () {
         self.setState({ 'activeSlide': activeSlide });
         self.setState({ 'lastActive': scrollDown ? activeSlide-- : activeSlide++ });
 
```

Comment:
Thanks for updating the dist files! I'll still do a transpile build, but it will likely look the exact same